src/components/DocumentationTopic/PrimaryContent/Declaration.vue

```diff
@@ -89,11 +90,18 @@ export default {
   computed: {
     /**
      * Show the captions of DeclarationGroup without changes
-     * when there are more than one declarations
+     * when there are multiple sets of platforms among the
+     * declarations.
      * @returns {boolean}
      */
     hasPlatformVariants() {
-      return this.declarations.length > 1;
+      const platforms = [];
```

Comment:
Could we possibly just create a computed property that is an object where the keys are the stringified list of platforms and the values are the actual lists of declarations mapped to those platforms? Likely the reason we weren't doing this before is because it was always expected to have a unique list of platforms per-declaration, and it sounds like that is no longer the case.

Reply:
After speaking with @hqhhuang I agree that since we need `platforms` and `tokens` in the same object for `DeclarationList` it may be easier and safer to keep it as it is.